SWLOR.Game.Server/Feature/AbilityDefinition/ForceHealAbilityDefinition.cs

```diff
@@ -0,0 +1,101 @@
+﻿//using Random = SWLOR.Game.Server.Service.Random;
+using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.AbilityService;
+using static SWLOR.Game.Server.Core.NWScript.NWScript;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition
+{
+    public class ForceHealAbilityDefinition : IAbilityListDefinition
+    {
+        public Dictionary<Feat, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            ForceHeal1(builder);
+            ForceHeal2(builder);
+            ForceHeal3(builder);
+            ForceHeal4(builder);
+            ForceHeal5(builder);
+
+            return builder.Build();
+        }
+
+        private static void ForceHeal1(AbilityBuilder builder)
+        {
+            builder.Create(Feat.ForceLeap1, PerkType.ForceHeal)
+                .Name("Force Heal 1")
+                .HasRecastDelay(RecastGroup.ForceHeal, 60f)
+                .HasActivationDelay(2.0f)
+                .RequirementFP(2)
+                .IsConcentrationAbility(StatusEffectType.ForceHeal1)
+                .DisplaysVisualEffectWhenActivating()
+                .HasImpactAction((activator, target, level) =>
+                {
+                    StatusEffect.Apply(activator, target, StatusEffectType.ForceHeal1, 0.0f);
```

Comment:
With what way I set up the IsConcentrationAbility() call you actually don't need an impact action here. Assuming it's working as expected ;)

Reply:
done